### PR TITLE
BPMSPL-301: request to improve error message while uploading spreadsheet with multibyte name

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/BPMSPL-301

This PR fixes the ConcurrentChangePopup (the "Uploading..." box was blocking the user from clicking in the popup buttons).

### CODE REVIEW

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenter.java [hunk 1]
```diff
@@ -86,12 +85,14 @@
     public DecisionTableXLSEditorPresenter( final DecisionTableXLSEditorView baseView,
                                             final DecisionTableXLSResourceType decisionTableXLSResourceType,
                                             final DecisionTableXLSXResourceType decisionTableXLSXResourceType,
+                                            final BusyIndicatorView busyIndicatorView,
```
Comment (manstis) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenter.java]: If you're moving ```BusyIndicatorView``` to the constructor, could you do the same for ```Event<NotificationEvent>```?

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java [hunk 2]
```diff
@@ -91,11 +102,13 @@ public void setUp() throws Exception {
         presenter = new DecisionTableXLSEditorPresenter( view,
                                                          decisionTableXLSResourceType,
                                                          decisionTableXLSXResourceType,
-                                                         new ServiceMock() ) {
+                                                         busyIndicatorView, new ServiceMock()
```
Comment (manstis) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java]: I'd be nice to keep parameters split on different lines.

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java [hunk 3]
```diff
@@ -91,11 +102,13 @@ public void setUp() throws Exception {
...
+        ) {
             {
                 kieView = mock( KieEditorWrapperView.class );
-                versionRecordManager = mock( VersionRecordManager.class );
                 overviewWidget = mock( OverviewWidgetPresenter.class );
+                versionRecordManager = versionRecordManagerMock;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java]: You have to admire the ease it is to test sub-classes of ```KieEditor``` ;-) static initialiser blocks to set protected-level inherited fields in tests.. always nice to see!!

**Comment by manstis** [comment]
@karreiro A couple of minor comments (although it'd probably be nice to see use of ```@Inject``` consistent re: fields vs constructor.

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenter.java (hunk 1 above)
Comment (karreiro) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenter.java]: Totally. Thanks, Mike :)

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java (hunk 2 above)
Comment (karreiro) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java]: Totally. Thanks, Mike :)

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java (hunk 3 above)
Comment (karreiro) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java]: Always nice.. :')

**Comment by karreiro** [comment]
Thanks @manstis :)

**Comment by manstis** [comment]
+1 to merge.

**Comment by manstis** [comment]
Thanks @karreiro

**Pull request merged**